Makefile

```diff
@@ -168,7 +168,7 @@ endif
 #   and the game code requires it.
 #  -fvisibility=hidden to keep symbols hidden. This is
 #   mostly best practice and not really necessary.
-override CFLAGS += -std=gnu99 -fno-strict-aliasing -fwrapv -fvisibility=hidden
+override CFLAGS += -std=gnu11 -fno-strict-aliasing -fwrapv -fvisibility=hidden
```

Comment:
Why the switch to C11? Okay, most up to date compilers have C11 support these days, but there are some users who still play the game on old hardware and maybe old Linux distros. I don't want to break them without good reason.

Reply:
ok I m going to roll it back.